### PR TITLE
Lets go to london

### PR DESCRIPTION
**Pull request by mark-r-m**
Lets go to London!!
- Add versions of Main and Equality feed that connect to London brokers
- Add new Audit feed that connects to London brokers

### CODE REVIEW

**Review by steveburton4 — DISMISSED** [review]

File: k2hb_ec2_asg_common.tf [hunk 1]
```diff
@@ -42,6 +42,18 @@ locals {
...
+
+  k2hb_audit_tags_asg = merge(
+    local.common_tags,
+    {
+      Name         = "${local.k2hb_equality_consumer_name}-${local.environment}",
+      k2hb-version = var.k2hb_version,
+      AutoShutdown = local.k2hb_equality_asg_autoshutdown[local.environment],
+      SSMEnabled   = local.k2hb_equality_asg_ssmenabled[local.environment],
+      Inspector    = local.k2hb_equality_asg_inspector[local.environment],
+      Persistence  = "Ignore",
```
Comment (connorIW) [inline k2hb_ec2_asg_common.tf:54]: Being referenced by aws_autoscaling_group.k2hb_audit_london in k2hb_ec2_asg_audit_london.tf.
Is that correct? Audit referencing equality tags?

File: k2hb_ec2_asg_common.tf [hunk 2]
```diff
@@ -42,6 +42,18 @@ locals {
       Persistence  = "Ignore",
     }
   )
+
+  k2hb_audit_tags_asg = merge(
+    local.common_tags,
+    {
+      Name         = "${local.k2hb_equality_consumer_name}-${local.environment}",
```
Comment (connorIW) [inline k2hb_ec2_asg_common.tf:49]: This line should reference audit, not equality.

File: k2hb_ec2_asg_equality_london.tf [hunk 3]
```diff
@@ -0,0 +1,169 @@
+# London
+
+locals {
+  k2hb_equality_london_tags_asg = merge(
+    local.k2hb_equality_tags_asg,
```
Comment (connorIW) [inline k2hb_ec2_asg_equality_london.tf:5]: This local doesn't exist in locals.tf. Can you let me know where it is located please?

**Review by connorIW — DISMISSED** [review]

File: k2hb_ec2_asg_equality_london.tf (hunk 3 above)
Comment (mark-r-m) [reply k2hb_ec2_asg_equality_london.tf:5]: on the common asg file

**Review by connorIW — APPROVED** [review]

**Pull request merged**